modules/n1ql/pages/n1ql-language-reference/collectionops.adoc

```diff
@@ -424,8 +424,8 @@ Return Values:: The `EXISTS` operator evaluates to `TRUE` if the value is an arr
 *EXISTS Example 1:* Of the 1641 cities with an airport, search for all cities that have a landmark as well.
 
 ----
-SELECT DISTINCT city FROM `travel-sample` AS l WHERE type = "landmark"
-AND EXISTS (SELECT city FROM `travel-sample` AS a WHERE type = "airport");
+SELECT DISTINCT city FROM `travel-sample`.inventory.landmark AS l
+WHERE EXISTS (SELECT city FROM `travel-sample`.inventory.airport AS a );
 ----
 
 This results in the 626 cities that contain both an airport and a landmark.
```

Comment:
```suggestion
This results in 255 cities that contain hotels with reviews.
```

Reply:
Please also replace the query results (lines 434 to 446) with the updated results:
```
[
  {
    "city": "Medway"
  },
  {
    "city": "Giverny"
  },
  {
    "city": "Glasgow"
  },
  {
    "city": "Highland"
  },
  ```